src/main/scala/dpla/ingestion3/entries/ingest/IngestRemap.scala

```diff
@@ -0,0 +1,73 @@
+package dpla.ingestion3.entries.ingest
+
+import dpla.ingestion3.confs.{CmdArgs, Ingestion3Conf}
+import dpla.ingestion3.entries.ingest.EnrichEntry.executeEnrichment
+import dpla.ingestion3.entries.ingest.JsonlEntry.executeJsonL
+import dpla.ingestion3.entries.ingest.MappingEntry.executeMapping
+import dpla.ingestion3.entries.reports.ReporterMain.executeAllReports
+import dpla.ingestion3.utils.Utils
+import org.apache.spark.SparkConf
+
+/**
+  * Single entry point to run harvest, mapping, enrichment, indexing jobs and all reports.
+  * Support running against an already harvested data set and running a fresh harvest
+  *
+  * Expects four parameters:
+  *   1)  --output  A base path to save the mapping, enrichment, json-l and report outputs
+  *   2)  --input   A path to the harvested data
+  *   3)  --conf    A path to the application configuration file
+  *   4)  --name    Provider short name
+  */
+object IngestRemap {
+
+  def main(args: Array[String]): Unit = {
+
+    // Read in command line args.
+    val cmdArgs = new CmdArgs(args)
+
+    val baseDataOut = cmdArgs.getOutput()
+    val confFile = cmdArgs.getConfigFile()
+    val shortName = cmdArgs.getProviderName()
+
+    // Outputs
+    val harvestDataOut = cmdArgs.getInput()
+    val mapDataOut = baseDataOut+"/mapped"
+    val enrichDataOut = baseDataOut+"/enriched"
+    val jsonlDataOut = baseDataOut+"/json-l"
+    val baseRptOut = baseDataOut+"/reports"
+
+    // Get logger
+    val logger = Utils.createLogger("ingest", shortName)
+
+    // Load configuration from file.
+    val i3Conf = new Ingestion3Conf(confFile, Some(shortName))
+    val conf = i3Conf.load()
+
+    // Read spark master property from conf, default to 'local[1]' if not set
+    val sparkMaster = conf.spark.sparkMaster.getOrElse("local[1]")
+
+    val sparkConf = new SparkConf()
+      .setAppName(s"Mapping: $shortName")
+      .set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
+      .set("spark.kryoserializer.buffer.max", "200")
+      .setMaster(sparkMaster)
+
```

Comment:
Don't you need to execute the harvest before you execute the mapping?  Does that happen somewhere else?

Reply:
Harvest needs to be run on its own.